examples/bench/README.md

```diff
@@ -0,0 +1,12 @@
+
+# Tsh Bench
+
+
+## Linear Benchmark Generator
+
+### What is this
+A linear generator generates benchmarks between a lower and upper bound using a fixed step as configured by the user. 
+
+
+### Use case
+Linear generators are useful when benchmarking setups with understood performance profiles or generating graphs for user-facing materials.
```

Comment:
Can you add in more instructions for how to use `tsh bench` and how someone can create graphs etc.